db/migrate/20210308053346_create_students.rb

```diff
@@ -0,0 +1,14 @@
+class CreateStudents < ActiveRecord::Migration
+  def change
+    create_table :students do |t|
+      t.string :first_name
+      t.string :middle_name
+      t.string :last_name
+      t.string :email
+      t.date :birth_date
+      t.string :gender
```

Comment:
is it possible to get rid of `null` for some of these fields? ie will they all be required when creating a student?
getting rid of null columns reduces `nil` leaking into code and having to cater for it.

```suggestion
      t.string :first_name, null: false, default: ""
      t.string :middle_name, null: false, default: ""
      t.string :last_name, null: false, default: ""
      t.string :email, null: false
      t.date :birth_date, null: false
      t.string :gender, null: false
```